src/NodaTime.Demo/OffsetDemo.cs

```diff
@@ -62,6 +62,19 @@ public void ConstructionFromTimeSpan()
             Assert.AreEqual(5400, offset.Seconds);
         }
 
+        [Test]
+        public void Max()
+        {
+            var offset = Offset.FromSeconds(10);
+            var offset2 = Offset.FromSeconds(20);
+            var expected = Offset.FromSeconds(20);
+
+            var actual = Snippet.For(Offset.Max(offset, offset2));
+
+            Assert.Less(offset, offset2);
```

Comment:
Please remove this - we translate just a very few Assert methods into Console.WriteLine - the idea is that the snippets aren't shown as if they're tests at all. I'd expect to see something like:

**Snippet**

```csharp
var smaller = Offset.FromHours(1);
var larger = Offset.FromHours(3);

var max = Offset.Max(smaller, larger);
Console.WriteLine(max);
```

**Output**

```text
+03
```